src/builtins/Transformers.jl

```diff
@@ -288,12 +289,35 @@ julia> transform(fit!(machine(select2, X)), X)
  nominal = CategoricalValue{String,UInt32}["Your father", "he", "is"],)
 
 ```
-
 """
-@with_kw_noshow mutable struct FeatureSelector <: Unsupervised
+mutable struct FeatureSelector <: Unsupervised
     # features to be selected; empty means all
-    features::Union{Vector{Symbol}, Function}=Symbol[]
-    ignore::Bool = false # features to be ignored
+    features::Union{Vector{Symbol}, Function}
+    ignore::Bool # features to be ignored
+    
+    # keyword constructor
+    function FeatureSelector(
+        ;
+        features::Union{AbstractVector{Symbol}, Function}=Symbol[],
+        ignore::Bool=false
+    )
+        transformer = new(features, ignore)
+        message = MLJBase.clean!(transformer)
+        isempty(message) || throw(ArgumentError(message))
+        return transformer
+    end
+end
```

Comment:
According to the [API docs](https://alan-turing-institute.github.io/MLJ.jl/dev/adding_models_for_general_use/#New-model-type-declarations-and-optional-clean!-method-1) you should not define internal constructors for MLJ models. Same for your changes to Standardizer below (which should not really be in this PR). 

Okay, this is new. Is there a reason you are making keyword constructors *internal* constructors? This is not in the API